protocol/0093-CFUT-product_builtin_capped_future.md

```diff
@@ -0,0 +1,63 @@
+
+- When the reference of a pegged sell order moves so that the implied order price is higher than `max_price` the implied order price gets capped at. `max_price` (<a name="0093-CFUT-003" href="#0093-CFUT-003">0093-CFUT-003</a>)
+
+- When market is setup to use oracle based mark price and the value received from oracle is less than `max_price` then it gets used as is and mark-to-market flows are calculated according to that price. (<a name="0093-CFUT-004" href="#0093-CFUT-004">0093-CFUT-004</a>)
+
+- When market is setup to use oracle based mark price and the value received from oracle is greater than `max_price` then it gets capped at `max_price` and mark-to-market flows are calculated according to `max_price`. (<a name="0093-CFUT-005" href="#0093-CFUT-005">0093-CFUT-005</a>)
+
+- When the final settlement price candidate received from the oracle is less than or equal to `max_price` and `binary_settlement` flag is set to `false` it gets used as is and the final cashflows are calculated according to that price. (<a name="0093-CFUT-006" href="#0093-CFUT-006">0093-CFUT-006</a>)
+
+- When the final settlement price candidate received from the oracle is greater than  `max_price` the value gets ignored, next a value equal to `max_price` comes in from the settlement oracle and market settles correctly. The market behaves in this way irrespective of how `binary_settlement` flag is set. (<a name="0093-CFUT-007" href="#0093-CFUT-007">0093-CFUT-007</a>)
+
+- When the final settlement price candidate received from the oracle is greater than `0` and less than  `max_price` and `binary_settlement` flag is set to `true` the value gets ignored, next a value of `0` comes in from the settlement oracle and market settles correctly. (<a name="0093-CFUT-008" href="#0093-CFUT-008">0093-CFUT-008</a>)
+
+- When market is setup with `fully_collateralised = true` option and it has parties with open positions settling it at a price of `max_price` works correctly and the sum of all final settlement cashflows equals 0 (loss socialisation does not happen). Assuming general account balances of all parties were `0` after opening the positions and all of their funds were in the margin accounts: long parties end up with balances equal to `position size * max_price` and short parties end up with `0` balances. (<a name="0093-CFUT-009" href="#0093-CFUT-009">0093-CFUT-009</a>)
+
+- When market is setup with `fully_collateralised = true` option and it has parties with open positions settling it at a price of `0` works correctly and the sum of all final settlement cashflows equals 0 (loss socialisation does not happen). Assuming general account balances of all parties were `0` after opening the positions and all of their funds were in the margin accounts: short parties end up with balances equal to `abs(position size) * max_price` and long parties end up with `0` balances. (<a name="0093-CFUT-010" href="#0093-CFUT-010">0093-CFUT-010</a>)
```

Comment:
TODO: add scenarious for the mark price being 0 and max price during trading and no close outs occurring

Reply:
Added these now